versioned_docs/version-2.0.0/quickstart/fastapi-sql.md

```diff
@@ -0,0 +1,617 @@
+---
+id: samples-fastapi
+title: Sample Student Data CRUD App (FastAPI)
+description: The following sample app showcases how to use the FastAPI framework and the Keploy Platform.
+tags:
+  - FastAPI Framework
+  - Postgres
+  - SQL
+keyword:
+  - FastAPI Framework
+  - Postgres
+  - SQL
+  - Python
+  - API Test generator
+  - Auto case generation
+---
+
+# Introduction
+
+🪄 Dive into the world of User CRUD Apps and see how seamlessly Keploy integrated with [FastAPI](https://fastapi.tiangolo.com/) and [PostgreSQL](https://www.postgresql.org/). Buckle up, it's gonna be a fun ride! 🎢
+
+## Pre-Requisite 🛠️
+
+- Install WSL (`wsl --install`) for <img src="/docs/img/os/windows.png" alt="Windows" width="3%" /> Windows.
+- Install Colima( `brew install colima && colima start` ) for <img src="/docs/img/os/macos.png" alt="MacOS" width="3%" /> MacOs.
+
+## Get Started! 🎬
+
+## Setup the PostgreSQL Database 📦
+
+  Create a docker network, run -
+
+  ```bash
+  docker network create backend
+  ```
+
+  Start the Postgres instance using the `docker-compose` file-
+
+  ```bash
+  docker run -p 5432:5432 -e POSTGRES_PASSWORD=postgres -d --network backend --name mypostgres postgres
+  ```
+
+  ```bash
+  docker build -t fastapi-app:1.0 .
+  ```
+
+## Clone the sample Student Data CRUD app 🧪
+
+```bash
+git clone https://github.com/keploy/samples-python.git && cd samples-python/fastapi-postgres
+```
```

Comment:
Which one of is for the fastapi-postgres sample app

Reply:
sorry, I'll have to remove the samples-fastapi.md and fastapi-sql.md is the one to be merged!!